python/lsst/ip/isr/isrTask.py

```diff
@@ -1198,7 +1216,7 @@ def readIsrData(self, dataRef, rawExposure):
     @pipeBase.timeMethod
     def run(self, ccdExposure, camera=None, bias=None, linearizer=None,
             crosstalk=None, crosstalkSources=None,
-            dark=None, flat=None, bfKernel=None, bfGains=None, defects=None,
+            dark=None, flat=None, ptc=None, bfKernel=None, bfGains=None, defects=None,
```

Comment:
This is a tricky one. Personally, I think the first line of args should read:
`def run(self, ccdExposure, *, camera=None...` so that we're mandating everything else to be a kwarg, because anyone trying to use this method trusting on position is asking for trouble. However, putting that in now might well break stuff in other places. So, you _could_ move this arg to the end of the list, or you could put in the `*` and be careful to run a full Jenkins including the CI packages etc to make sure nothing actually breaks. I'd prefer doing it that way, but understand if you want to just move this to the end of the list. But it's got to be one of the two in order not to potentially break people doing everything positionally.

Reply:
Quite curious what @czwa thinks is the right approach here. For functions with horrendous numbers of args we should be mandating `*` really, right?

Reply:
FWIW, the `*` option at least passes the `ip_isr` tests...

Reply:
(but `isrTask.py` has the 2nd worst coverage in the whole package)

Reply:
Putting `*` sounds reasonable to me, but I'll also wait for Chris to comment.  What other packages should be included in Jenkins? I usually use the defaults (`lsst_distrib lsst_ci`) + `ci_cpp_gen3`. I guess I can also do `ci_cpp_gen2`. Anything else?

Reply:
I think ci_hsc too, basically just thinking we should exercise anything/everything we have that actually runs isr to check (as best we can) that we're not breaking things in the wild. To be honest though, anyone who does have this break stuff for them was playing a dangerous game already I think, so I don't think it's _too_ big of a deal, but certainly worth running anything we can that uses isr.

Reply:
I usually add ci_hsc and ci_cpp_gen3 to the defaults.

As for the actual question, I think adding the * is probably worth the three extra characters.  I would hope no one is relying on position for this method, and the standard case has the arguments passed automatically from `runQuantum`.